sdk/test/common/empty_attributes_test.cc

```diff
@@ -0,0 +1,8 @@
+#include "opentelemetry/sdk/common/empty_attributes.h"
+
+#include <gtest/gtest.h>
+
+TEST(EmptyAttributesTest, EmptyTest)
+{
+  EXPECT_EQ((opentelemetry::sdk::GetEmptyAttributes()).size(), 0);
```

Comment:
As an additional tests, we could call `GetEmptyAttributes` two times, cast the `KeyValueIterable` to a `KeyValueIterableView` and then do a `memcmp` on both. This would test that we actually return the same map twice.

Reply:
Why is it necessary to cast to a `KeyValueIterableView`? That's already what `GetEmptyAttributes()` returns...

Reply:
Sorry, nevermind. For some reason I thought it's returning a `KeyValueIterable`.

Reply:
I've added a test that uses `memcmp` to compare two values returned by `GetEmptyAttributes()`.